wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -936,6 +938,7 @@ public androidx.work.Configuration getWorkManagerConfiguration() {
                 .build();
     }
 
+    @Deprecated(message = "Inject instead")
```

Comment:
`.getInstance()` calls really hinder testing. We should avoid using them and continue to migrate to injections instead.